backend/tests/expenses/test_expense_service.py

```
@@ -0,0 +1,144 @@
+import pytest
+from main import app # Adjusted import - Keep app import for context if needed, but TestClient is removed
```

Ruff (0.11.9)

`main.app` imported but unused

Remove unused import: `main.app`

(F401)